components/event-handler-notification/org.wso2.carbon.identity.event.handler.notification/pom.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  ~ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
```

Comment:
We can't update the year, only need to update the LLC part for the existing files. 
Check other places as well

Reply:
Fixed with 96bf61bf4c323ea0d42a24507385ef25f187ffee